README.md

```diff
@@ -35,7 +35,9 @@ Here's an example
 
 ### Send to dirname deck
 
-Similar to the above but will use the current directory structure as deck name.
+Similar to the above but will use the current directory structure as deck name. For example, you write a file named `/path/to/workspace/deck/subdeck/note.md`, it will be send to deck `deck::subdeck`.
+
```

Comment:
Ah I wasn’t aware of this, I assumed the file name also constituted as part of the deck, so it would be deck::subdeck::note. What happens when there’s multiple files on the same folder? How do they go to the same deck?

Reply:
I think the concept of deck in Anki is similar to the directory in the file system, while the concept of card is similar to the file, so I don't take the file name as a part of deck. But I didn't change the code of other parts, so users can still create one or multiple cards in one file through `##`.
when there’s multiple files on the same folder, they will be send to the same deck, because the param of function `relative` in `src/markdown/transformer.ts` don't contains of file name (use `dirname`).
So the actual situation is that I ignore the file name and only use `##` to identify each card like before.

Reply:
Ok sounds good to me